styles.css

```diff
@@ -401,3 +397,230 @@ textarea {
   text-decoration: none;
   margin-bottom: 3rem;
 }
+
+@media only screen and (min-width: 768px) {
+  .container {
+    width: 90%;
+    padding: 1rem;
+    margin: 0 auto;
+    background: none;
+  }
+
+  .headline {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 160px 0;
+    width: 90%;
+    height: 756px;
+    left: 260px;
+    top: 72px;
+    background: url(images/Header-desktop-img.png);
+    background-size: contain;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  .headline-title {
+    width: 920px;
+    height: 114px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 48px;
+    line-height: 60px;
+    text-align: center;
+    letter-spacing: 0.37px;
+    color: #172b4d;
+  }
+
+  .headline-subtitle {
+    width: 700px;
+    height: 96px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: #172b4d;
+    margin-top: -30px;
+  }
+
+  .work-container {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 80px 0;
+    gap: 48px;
+  }
+
+  .work-heading > h2 {
+    width: 450px;
+    height: 48px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    color: #172b4d;
+  }
+
+  .line {
+    display: none;
+  }
+
+  .work-sub-container {
+    display: flex;
+    flex-direction: column;
+    padding: 0;
+    gap: 24px;
+  }
+
+  .row1 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 0;
+    gap: 24px;
+  }
+
+  .row2 {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    padding: 4px 0;
+    gap: 24px;
+  }
+
+  .box {
+    height: none;
+  }
+
+  .title-card {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    gap: 24px;
+    width: 90%;
+    background-repeat: no-repeat;
+    background-position-y: bottom;
+  }
+
+  .about-me-subtitle {
+    width: 920px;
+    height: 96px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+    text-align: center;
+    color: #42526e;
+  }
+
+  .about-container {
+    display: flex;
+    gap: 24px;
+  }
+
+  /* form section */
+  .form-section {
+    margin: 0 auto;
+    width: 90%;
+    background: none;
+  }
+
+  .text-form {
+    display: flex;
+    padding: 80px 0;
+    gap: 25px;
+    width: 1171px;
+    height: 490px;
+    justify-content: center;
+    align-items: flex-start;
+    background: url(images/left-form-bg-img.svg) bottom left, url(images/left-center-form-bg-img.svg) bottom center, url(images/right-form-bg-img.svg) bottom right;
+    background-repeat: no-repeat;
+    background-position: cover;
+  }
```

Comment:
- [ ] Good job with your styling 👍 . Kindly make sure the background images are positioned towards the end of the webpage as seen in the Figma design 🙏 

Figma Design | Your Design
------------- | -------------
![image](https://user-images.githubusercontent.com/74814780/186772380-322e7e8d-9210-4a0d-924e-ba97afd7d7b4.png)  |  ![image](https://user-images.githubusercontent.com/74814780/186772526-ec049ce2-447a-48ae-bae4-3f3024ef1254.png)